core/shared/src/main/scala/zio/Runtime.scala

```diff
@@ -174,4 +200,59 @@ object Runtime {
     val environment = r
     val platform    = platform0
   }
+
+  lazy val default = Runtime((), Platform.default)
+
+  lazy val global = Runtime((), Platform.global)
+
+  /**
+   * Unsafely creates a `Runtime` from the default platform and a `ZLayer`
+   * whose resources will be allocated immediately, and not released until the
+   * `Runtime` is shut down or the end of the application.
+   *
+   * This method is useful for small applications and integrating ZIO with
+   * legacy code, but other applications should investigate using
+   * [[ZIO.provideLayer]] directly in their application entry points.
+   */
+  def unsafeDefault[R <: Has[_]](layer: ZLayer.NoDeps[Any, R]): Runtime.Managed[R] =
+    unsafeMake(layer, default.platform)
+
+  /**
+   * Unsafely creates a `Runtime` from Scala's global execution context and a
+   * `ZLayer` whose resources will be allocated immediately, and not released
+   * until the `Runtime` is shut down or the end of the application.
+   *
+   * This method is useful for small applications and integrating ZIO with
+   * legacy code, but other applications should investigate using
+   * [[ZIO.provideLayer]] directly in their application entry points.
+   */
+  def unsafeGlobal[R <: Has[_]](layer: ZLayer.NoDeps[Any, R]): Runtime.Managed[R] =
+    unsafeMake(layer, global.platform)
+
+  /**
+   * Unsafely creates a `Runtime` from a `ZLayer` whose resources will be
+   * allocated immediately, and not released until the `Runtime` is shut down
+   * or the end of the application.
+   *
+   * This method is useful for small applications and integrating ZIO with
+   * legacy code, but other applications should investigate using
+   * [[ZIO.provideLayer]] directly in their application entry points.
+   */
+  def unsafeMake[R <: Has[_]](layer: ZLayer.NoDeps[Any, R], platform: Platform): Runtime.Managed[R] = {
+    val runtime = Runtime((), platform)
+    val (environment, shutdown) = runtime.unsafeRun {
+      layer.build.reserve.flatMap {
+        case Reservation(acquire, release) =>
+          Ref.make(true).flatMap { finalize =>
+            val finalizer = () =>
```

Comment:
To be "ZIO best practice" here we should wrap the inner block in `UIO.effectSuspend` since we are calling `addShutdownHook` there. Won't make a difference here but could be useful for people perusing the code.

Reply:
Good point. Will fix.